css/projects.css

```diff
@@ -421,6 +421,49 @@ label, input[type='radio'] {
   color:rgba(0,0,0,0.67);
 }
 
+/* detail view */
+
+#lightbox-overlay {
+  background: rgba(0,0,0,.85);
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  z-index: 1000;
+  display: none;
+}
+
+#detail-view-wrapper {
+  position: fixed;
+  top: 20px;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+  margin: 0;
+  z-index: 100000;
+  display: none;
+}
+
+#close-control {
+  float: right;
+  background: #fff;
+  cursor: pointer;
+}
+
+#detail-view {
+  background: #fff;
+  margin: 100px 200px;
```

Comment:
The large margin here affects the detail view on mobile
![screen shot 2016-07-19 at 18 22 34](https://cloud.githubusercontent.com/assets/617994/16968675/596c194e-4dde-11e6-9e69-76b95d83ca63.png)